docs-src/tut-7-rpc.scrbl

```diff
@@ -183,6 +210,144 @@ invoke our @pyin{main} function.
+.................++++
+writing new private key to './tls/reach-server.key'
+Enter PEM pass phrase:
+Verifying - Enter PEM pass phrase:
+-----
+Signature ok
+subject=CN = reach-server
+Getting Private key
+Enter pass phrase for ./tls/reach-server.key:
+  ...done
+
+When invoking `reach server` you MUST supply the pass phrase you just
+used to generate your certificate as an environment variable, e.g.:
+
+  $ REACH_RPC_TLS_PASSPHRASE=ENTER-PASSPHRASE-HERE \
+    REACH_RPC_KEY=YOUR-PRESHARED-KEY \
+    ./reach server
+
+./tls
+├── reach-server.crt
+├── reach-server.csr
+├── reach-server.ext
+└── reach-server.key
+
+0 directories, 4 files
+}
+
+@(hrule)
+
+With those created, now we launch the server:
+
+@verbatim{
+$  REACH_RPC_TLS_PASSPHRASE=ENTER-PASSPHRASE-HERE \
+   REACH_RPC_KEY=YOUR-PRESHARED-KEY \
+   ./reach server
+
+===============================
+using rpc key:
+REACH_RPC_KEY=YOUR-PRESHARED-KEY
+===============================
+
+Verifying knowledge assertions
+Verifying for generic connector
+  Verifying when ALL participants are honest
+  Verifying when NO participants are honest
+  Verifying when ONLY "Alice" is honest
+  Verifying when ONLY "Bob" is honest
+Checked 93 theorems; No failures!
+Creating network "rps-python_default" with the default driver
+Creating rps-python_ethereum-devnet_1 ... done
+
+> @"@reach-sh/rpc-server@" app /app
+> node --experimental-modules --unhandled-rejections=strict index.mjs
+
+(node:26) ExperimentalWarning: The ESM module loader is experimental.
+}
+
+The server is now awaiting connections.
+
+@(hrule)
+
+Open another terminal in your @tt{rps-python} directory and type the following
+to play a game of @|RPS|:
+
+@margin-note{
+Alternatively, if you would prefer to supply configuration as arguments to the
+@pyin{mk_rpc} function at runtime instead of using environment variables, this
+also works:
+
+@verbatim{
+rpc, rpc_callbacks = \
+    mk_rpc(dict(
+  host='127.0.0.1',
+  port='3000',
+  verify='0',
+  timeout='7',
+  key='YOUR-PRESHARED-KEY',
+))
+}
+
+
+You may choose whichever best suits your use case.
+
+Combining both methods is also possible, but settings supplied as arguments to
+@pyin{mk_rpc} are given priority over settings defined as environment
+variables.
+
+Read @seclink{ref-backends-rpc-opts} for more details.
+}
+
+@verbatim{
+$ . ./venv/bin/activate
+$ REACH_RPC_SERVER=127.0.0.1 \
+  REACH_RPC_PORT=3000 \
+  REACH_RPC_KEY=YOUR-PRESHARED-KEY \
+  REACH_RPC_TLS_REJECT_UNVERIFIED=0 \
+  python3 -u ./index.py
+
+*** Warning! TLS verification disabled! ***
+
+ This is highly insecure in Real Life™ applications and must
+ only be permitted under controlled conditions (such as
+ during development).
+
+Bob accepts the wager of 5
+Alice played Rock
+Bob played Paper
+Bob saw outcome Bob wins
+Alice saw outcome Bob wins
+Alice went from 10 to 4.9999
+  Bob went from 10 to 14.9999
+}
+
+You may replay as many times as you would like by pressing @tt{"up"} once then
```

Comment:
Make a point that this is the same output as in the normal tutorial, but you just used Python, go you!